editComment.php

```diff
@@ -0,0 +1,68 @@
+<?php 
+    include('init.php');
+    if($_SERVER['REQUEST_METHOD']=="POST"){
+
+        //Initializing the variables
+		$messages=array();
+        $response=array();
+
+        $user_id = clean($_POST['user_id']);
+        $user_name = clean($_POST['user_name']);
+        $user_img = $_POST['user_img'];
+        $post_id = clean($_POST['post_id']);
+        $comment_id = clean($_POST['comment_id']);
+        $body = clean($_POST['body']);
+        if(isset($_POST['images']))
+        $images = clean($_POST['images']);
+        else $images = serialize(array());
+        $upvotes = serialize(array());
+        $downvotes = serialize(array());
+        if(empty($body)){
+            $messages[] = "Message can not be empty.";
+            $response['status'] = 408;
+        }else{
+
+
+            $sql = "SELECT `id` FROM threads WHERE post_id=$post_id and id=$comment_id";
+            $result = query($sql);
+            
+            if (row_count($result) == 1){
+
+
+            $sql = " UPDATE `threads`(`post_id`,`user_name`,`user_img`, `thread_time`, `thread_body`, `thread_imgs`, `user_id`,`upvotes_ids`,`downvotes_ids`) VALUES ($post_id, '$user_name','$user_img', NOW(), '$body', '$images', $user_id, '$upvotes', '$downvotes')";
```

Comment:
user should not be able to update post id, username, user_img, user_img, Upvotes, and downvotes. We also do not want to update the thread_time.